src/Uno.UI/UI/Xaml/Controls/ScrollContentPresenter/ScrollContentPresenter.cs

```diff
@@ -200,22 +200,19 @@ private void PointerWheelScroll(object sender, Input.PointerRoutedEventArgs e)
 				{
 					// TODO: Handle zoom https://github.com/unoplatform/uno/issues/4309
 				}
-				else if (!canScrollVertically || properties.IsHorizontalMouseWheel || e.KeyModifiers == VirtualKeyModifiers.Shift)
```

Comment:
Is the Shift modifier not needed anymore?

Reply:
I'm not sure. I tested on WinUI and shift doesn't seem to do anything either horizontally or vertically, so I'm not sure why that was in there in the first place. A quick search also didn't yield anything.

Reply:
If you enable vertical **and** horizontal scrolling, holding the shift key will allow you to scroll horizontally with the vertical wheel of you mouse. This is a common usage, not only with WinUI but also windows, web browsers etc.